### ISSUE TITLE
windows ??

### ISSUE THREAD

**Issue by zachysaur** [myshell-ai/MeloTTS#73]
how to install on windows

**Comment by Nenesh** [comment]
If you don't want to use training, this should work (training don't work for me on windows without using WSL or Docker, "torchrun: command not found") :

git clone https://github.com/myshell-ai/MeloTTS.git
cd MeloTTS
conda create -n melotts python=3.10
conda activate melotts
pip install -e .
python -m unidic download

**Comment by zachysaur** [comment]
if i dont want to use docker and wsl

**Comment by zachysaur** [comment]
how i can train some voice?

**Comment by Nenesh** [comment]
Just install like i say, and test training.
If training doesn't work, i don't know.

**Comment by arkilis** [comment]
No module named unidic

**Comment by YodaGitMaster** [comment]
Followed these steps:
```
git clone https://github.com/myshell-ai/MeloTTS.git
cd MeloTTS
conda create -n melotts python=3.10
conda activate melotts
pip install -e .
python -m unidic download
```

than used

`melo-ui`

Got error:
```
LookupError:
**********************************************************************
  Resource averaged_perceptron_tagger_eng not found.
  Please use the NLTK Downloader to obtain the resource:

  >>> import nltk
  >>> nltk.download('averaged_perceptron_tagger_eng')

  For more information see: https://www.nltk.org/data.html

  Attempted to load taggers/averaged_perceptron_tagger_eng/

  Searched in:
    - 'C:\\Users\\gamin/nltk_data'
    - 'C:\\Users\\gamin\\miniconda3\\envs\\melotts\\nltk_data'
    - 'C:\\Users\\gamin\\miniconda3\\envs\\melotts\\share\\nltk_data'
    - 'C:\\Users\\gamin\\miniconda3\\envs\\melotts\\lib\\nltk_data'
    - 'C:\\Users\\gamin\\AppData\\Roaming\\nltk_data'
    - 'C:\\nltk_data'
    - 'D:\\nltk_data'
    - 'E:\\nltk_data'
**********************************************************************
```

**Comment by louistiti** [comment]
@YodaGitMaster to fix this, you can download the NLTK data first:

Run the Python interpreter:
```bash
python
```

Import NLTK and download the dataset:
```
import nltk
nltk.download('averaged_perceptron_tagger_eng')
```

Or you can also try this directly:

```bash
python -m nltk.downloader averaged_perceptron_tagger_eng
```

If you still have some missing files, you can directly download all the data via by replacing `averaged_perceptron_tagger_eng` with `all`. But this will download many data that you might not need. You can also try with `popular` instead of `all`.

More info here: https://www.nltk.org/data.html.

Hope it helps! :smiley:

**Comment by khooinguyeen** [comment]
C:\Users\Nguyen\anaconda3\envs\melotts\lib\site-packages\torch\nn\utils\weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
checkpoint.pth:   0%|                                                                                                           | 0.00/208M [00:00<?, ?B/s]Error while downloading from https://cdn-lfs-us-1.hf.co/repos/8a/1e/8a1ef674c4775bc19ceb0e1b85c7136ea7f9b0157416ed9a7087ac84f4160675/acd278040eaf9536908e2b965273df5a731c44d8f0da66cc5fed7972772ed23c?response-content-disposition=inline%3B+filename*%3DUTF-8%27%27checkpoint.pth%3B+filename%3D%22checkpoint.pth%22%3B&Expires=1737112285&Policy=eyJTdGF0ZW1lbnQiOlt7IkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTczNzExMjI4NX19LCJSZXNvdXJjZSI6Imh0dHBzOi8vY2RuLWxmcy11cy0xLmhmLmNvL3JlcG9zLzhhLzFlLzhhMWVmNjc0YzQ3NzViYzE5Y2ViMGUxYjg1YzcxMzZlYTdmOWIwMTU3NDE2ZWQ5YTcwODdhYzg0ZjQxNjA2NzUvYWNkMjc4MDQwZWFmOTUzNjkwOGUyYjk2NTI3M2RmNWE3MzFjNDRkOGYwZGE2NmNjNWZlZDc5NzI3NzJlZDIzYz9yZXNwb25zZS1jb250ZW50LWRpc3Bvc2l0aW9uPSoifV19&Signature=BxbHteb9xh0PuKJCypBwyY2tf2L6EHoTC9t%7EpJmfvNWKGiTHKekO7gy2Yyt0tMVEbR6vKjMgqckLiF8QqG2KRmUcT8k3rukgbqhyksXkpKto%7EnCnZN3CqY8zleTkjqP8SsATMba65T5m2d1ctDudCj1PyF7yKnOemLIwtlnoSYufV4Sroa0FcUXkTDjE8eCc-nN12QQ8EUSqUyNF1iwy6tb3GCLroTEqNC-eEfvLMbJK2RnhZ6d0J056Df0KdHeuSCnzEsCSWaVF%7EN4IeGqScKPqyxqQ1RqlrSXxcbYv6PzHzDrjOwOuRQbXs%7EuUHZvR8YhJxvDa8a8yf9p36ppfPw__&Key-Pair-Id=K24J24Z295AEI9: HTTPSConnectionPool(host='cdn-lfs-us-1.hf.co', port=443): Read timed out.
Trying to resume download...

checkpoint.pth:   0%|                                                                                                           | 0.00/208M [00:00<?, ?B/s] 



Hi, I followed the steps in the comments above but when I run melo-ui, it stucks at 0% when downloading checkpoint.pth (My internet connection is strong). Do you guys know how to fix this?

**Comment by Nikoluque** [comment]
How can i fix this error?: Unable to create process using 'C:\Users\Admin\miniconda3\python.exe "C:\Users\Admin\miniconda3\Scripts\[pip-script.py](javascript:void(0);)" install -e .'
